model_servers/llamacpp_python/src/run.sh

```diff
@@ -11,7 +11,7 @@ if [ ${MODEL_PATH} ]; then
         --port ${PORT:=8001} \
         --n_gpu_layers ${GPU_LAYERS:=0} \
         --clip_model_path ${CLIP_MODEL_PATH:=None} \
-        --chat_format ${CHAT_FORMAT:="llama-2"} \
+        --chat_format ${CHAT_FORMAT:=None} \
```

Comment:
I'm afraid that will cause an error at execution time

Reply:
Worked fine for me:
```
$ make build && make run
podman build --squash-all  --build-arg PORT=8001 -t quay.io/ai-lab/model_servers/llamacpp_python:latest . -f base/Containerfile
STEP 1/10: FROM registry.access.redhat.com/ubi9/python-311:1-62.1716478620
...
Writing manifest to image destination
--> 537a5ca8e8c9
[Warning] one or more build args were not consumed: [PORT]
Successfully tagged quay.io/ai-lab/model_servers/llamacpp_python:latest
537a5ca8e8c9334451fc9dc8e31be97bc176738a1e3de4b9146417d802c2adb6
cd ../../models && \
	podman run -it \
		-d \
		-p 8001:8001 \
		-v ./granite-7b-lab-Q4_K_M.gguf:/locallm/models/granite-7b-lab-Q4_K_M.gguf:ro \
		-e MODEL_PATH=/locallm/models/granite-7b-lab-Q4_K_M.gguf \
		-e HOST=0.0.0.0 \
		-e PORT=8001 \
		 \
		quay.io/ai-lab/model_servers/llamacpp_python:latest
21157b18f1f34f1bc154281332295d873a5329a128113149e6f900bcafcea26a
$ podman ps -a
CONTAINER ID  IMAGE                                                COMMAND     CREATED         STATUS         PORTS                   NAMES
21157b18f1f3  quay.io/ai-lab/model_servers/llamacpp_python:latest              38 seconds ago  Up 39 seconds  0.0.0.0:8001->8001/tcp  happy_archimedes
$ podman logs 21157b18f1f3
podman logs 21157b18f1f3
llama_model_loader: loaded meta data with 25 key-value pairs and 291 tensors from /locallm/models/granite-7b-lab-Q4_K_M.gguf (version GGUF V3 (latest))
llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.
...
INFO:     Started server process [2]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8001 (Press CTRL+C to quit)
```

Let me know if I am missing something though.